### PR TITLE
[IMP] hr_holidays: Improve diplayed information

### PR DESCRIPTION
**Pull request by nit-odoo**
LINKS
PR: #58168
Task-Id: 2314838

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_holidays/models/hr_leave.py [hunk 1]
```diff
@@ -856,7 +856,7 @@ def _create_resource_leave(self):
         :returns: created `resource.calendar.leaves`
         """
         vals_list = [{
-            'name': leave.name,
+            'name': "%s: Time Off" % (leave.employee_id.name),
```
Comment (kba-odoo) [inline addons/hr_holidays/models/hr_leave.py:859]:
```suggestion
            'name': _("%s: Time Off", leave.employee_id.name),
```

File: addons/hr_work_entry_holidays/models/hr_leave.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ def _create_resource_leave(self):
             contract = leave.employee_id.sudo()._get_contracts(leave.date_from, leave.date_to, states=['open'])
             if contract and contract.resource_calendar_id != leave.employee_id.resource_calendar_id:
                 resource_leave_values += [{
-                    'name': leave.name,
+                    'name': "%s: Time Off" % (leave.employee_id.name),
```
Comment (kba-odoo) [inline addons/hr_work_entry_holidays/models/hr_leave.py:37]:
```suggestion
                    'name': _("%s: Time Off", leave.employee_id.name),
```

**Review by kba-odoo — CHANGES_REQUESTED** [review]
Use `_()` to have the string translated

File: addons/hr_work_entry_holidays/models/hr_leave.py (hunk 2 above)
Comment (mba-odoo) [reply addons/hr_work_entry_holidays/models/hr_leave.py]: @kba-odoo , Done the changes. Thanks

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**